version/version.go

```diff
@@ -23,7 +23,8 @@ var Long = ""
 
 // Short is a short version number for this build, of the form
 // "x.y.z" for builds stamped in the usual way (see
-// build_dist.sh in the root) or, for binaries built by hand with the                                                                            // go tool, it's like Long's dev form, but ending at the date part,
+// build_dist.sh in the root) or, for binaries built by hand with the
```

Comment:
Spurious change? I can't tell what's different.

Reply:
The "split" github view hides it with line wrapping, but there's a missing newline here being inserted.  More obvious if you look at the current file in main: https://github.com/tailscale/tailscale/blob/27f36f77c364562bc1ca660c241291e23606e54d/version/version.go#L26